package exceptions;

public class DuplicatedTaskException {
    public DuplicatedTaskException() {
        System.out.println("* Failed! This task has been added to the table. *");
    }
}
